src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -129,7 +129,7 @@ public class Options : ISupportBoosterParameterFactory
                 public double MinChildWeight = 0.1;
 
                 [Argument(ArgumentType.AtMostOnce,
-                    HelpText = "Subsample frequency. 0 means no subsample. "
+                    HelpText = "Subsample frequency for bagging. 0 means no subsample. "
                     + "If subsampleFreq > 0, it will use a subset(ratio=subsample) to train. And the subset will be updated on every Subsample iteratinos.")]
                 [TlcModule.Range(Min = 0, Max = int.MaxValue)]
                 public int SubsampleFreq = 0;
```

Comment:
>SubsampleFreq [](start = 27, length = 13)

What does it mean if we set 5? Please also rename it to `SamplingFrequency`

Reply:
This is the frequency at which the subsampling happens -- so in the example you mention, 5 would be the iteration. The 20% is controlled by setting the Subsample parameter to .20.

---
In reply to: [261474003](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261474003) [](ancestors = 261474003)

Reply:
Could you make this example a part of `SubsampleFrequency`'s doc string?

---
In reply to: [261780306](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261780306) [](ancestors = 261780306,261474003)